mobile/library/cc/engine_builder.cc

```diff
@@ -189,6 +189,17 @@ EngineBuilder& EngineBuilder::enableH2ExtendKeepaliveTimeout(bool h2_extend_keep
   return *this;
 }
 
+EngineBuilder& EngineBuilder::addRtdsLayer(const std::string& layer_name) {
+  custom_layers_ = fmt::format(rtds_layer_insert, layer_name, layer_name);
+  return *this;
+}
+EngineBuilder& EngineBuilder::setAggregatedDiscoveryService(const std::string& api_type,
```

Comment:
`api_type` will always be `GRPC`, at least for now, so we can further simply it

Reply:
Ali changed his mind